lms/djangoapps/instructor_task/tests/test_api.py

```diff
@@ -140,38 +145,43 @@ def test_submit_reset_all_with_long_url(self):
     def test_submit_delete_all_with_long_url(self):
         self._test_submit_with_long_url(submit_delete_problem_state_for_all_students)
 
-    def _test_submit_task(self, task_function, student=None):
+    @ddt.data(
+        (submit_rescore_problem_for_all_students, 'rescore_problem'),
+        (submit_rescore_problem_for_all_students, 'rescore_problem_if_higher', dict(only_if_higher=True)),
+        (submit_rescore_problem_for_student, 'rescore_problem', dict(student=True)),
+        (submit_rescore_problem_for_student, 'rescore_problem_if_higher', dict(student=True, only_if_higher=True)),
+        (submit_reset_problem_attempts_for_all_students, 'reset_problem_attempts'),
+        (submit_delete_problem_state_for_all_students, 'delete_problem_state'),
+        (submit_rescore_entrance_exam_for_student, 'rescore_problem', dict(student=True)),
```

Comment:
Nit: I dislike using the dict keyword this way, and prefer dict literals.  This way introduces more line noise, can be shadowed by careless devs, looks unidiomatic, and is slightly slower.  The differences can be seen here:

```python
>>> import dis
>>> def return_dict():
...     return dict(a=1, b=2)
... 
>>> dis.dis(return_dict)
  2           0 LOAD_GLOBAL              0 (dict)
              3 LOAD_CONST               1 ('a')
              6 LOAD_CONST               2 (1)
              9 LOAD_CONST               3 ('b')
             12 LOAD_CONST               4 (2)
             15 CALL_FUNCTION          512
             18 RETURN_VALUE        
>>> def return_dict_literal():
...     return {"a": 1, "b": 2}
... 
>>> dis.dis(return_dict_literal)
  2           0 BUILD_MAP                2
              3 LOAD_CONST               1 (1)
              6 LOAD_CONST               2 ('a')
              9 STORE_MAP           
             10 LOAD_CONST               3 (2)
             13 LOAD_CONST               4 ('b')
             16 STORE_MAP           
             17 RETURN_VALUE        
```